tests/components/lifx/test_light.py

```diff
@@ -71,7 +71,11 @@ async def test_light_unique_id(hass: HomeAssistant) -> None:
     device = device_registry.async_get_device(
         identifiers=set(), connections={(dr.CONNECTION_NETWORK_MAC, SERIAL)}
     )
-    assert device.identifiers == {(DOMAIN, SERIAL)}
+    assert device.identifiers == {
+        (DOMAIN, SERIAL),
+        (DOMAIN, f"{SERIAL}_identify"),
+        (DOMAIN, f"{SERIAL}_restart"),
+    }
```

Comment:
The device shouldn't get new identifiers for the entities since the identifiers are for the device itself and not the entities.